services/community-backend/extensions/users-permissions/controllers/Auth.js

```diff
@@ -0,0 +1,304 @@
+const _ = require('lodash');
+const { sanitizeEntity } = require('strapi-utils');
+const axios = require('axios');
+const crypto = require('crypto');
+const emailRegExp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const formatError = error => [
+  { messages: [{ id: error.id, message: error.message, field: error.field }] },
+];
+
+
+
+module.exports = {
+  async forgotPassword(ctx) {
+    let { email, token } = ctx.request.body;
+    const gres = await axios.post(`https://www.google.com/recaptcha/api/siteverify?secret=6LdLJXAaAAAAALsfpslEdIBHpW-OYKPuUwFT2lbY&response=${token}`)
```

Comment:
We should move this secret to an ENV var

Reply:
@VargaElod23 handled pulling config from ENV and I have added `Secret` to helm-chart.